docker-compose.yaml

```diff
@@ -7,7 +7,7 @@ version: '3.7'
 services:
   gooddata-cn-ce:
     # Use the dev_latest tag which points to the latest development
-    image: registry.gitlab.com/gooddata/gdc-nas/gooddata-cn-ce:stable
+    image: 020413372491.dkr.ecr.us-east-1.amazonaws.com/nas/gooddata-cn-ce:stable
```

Comment:
The comment above talks about the latest tag, but stable tag is used.
Please, explain shortly why we use stable.